src/Query/Builder.php

```diff
@@ -1651,16 +1654,31 @@ private function aliasIdForQuery(array $values): array
         return $values;
     }
 
-    private function aliasIdForResult(array $values): array
+    private function aliasIdForResult(array|object $values): array|object
```

Comment:
You could use an `@template` annotation to indicate that the return type will be the same as the parameter type.

Reply:
Yes, but it's a private function and doesn't pose any downstream problems.

Reply:
Fixed.